.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,34 @@
+settings:
+    # Jira project key to create the issue in
+  jira_project_key: "KU"
+
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done
+
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  components:
+    - K8s snap
```

Comment:
```suggestion
    - K8s dqlite
```

Maybe it's better to differantiate this

Reply:
Yes there is no such component. In our cards we rarely specify the component field. We can remove it or add microk8s snap there as well since both use k8s-dqlite.

Reply:
Makes senes, I've added "Microk8s snap" as well.